1650115975045	Sync.LogManager	DEBUG	Flushing file log
1650115975048	Sync.LogManager	DEBUG	Log cleanup threshold time: 1649251975048
1650115975063	Sync.LogManager	DEBUG	Done deleting files.
1650117662826	Sync.SyncScheduler	DEBUG	Going to sleep, doing a quick sync.
1650117662827	Sync.Service	DEBUG	User-Agent: LibreWolf/99.0 (Linux x86_64) FxSync/1.101.0.20220410100435.desktop
1650117662827	Sync.Service	INFO	Starting sync at 2022-04-16 14:01:02 in browser session H4WfuUmnwKie
1650117662827	Sync.SyncScheduler	DEBUG	Clearing sync triggers and the global score.
1650117662829	Sync.Status	INFO	Resetting Status.
1650117662830	Sync.Status	DEBUG	Status.service: error.login.failed => success.status_ok
1650117662830	FirefoxAccounts	DEBUG	FxAccountsProfileClient: Requested profile
1650117662830	FirefoxAccounts	DEBUG	getOAuthToken enter
1650117662830	FirefoxAccounts	TRACE	getCachedToken returning cached token
1650117662830	FirefoxAccounts	DEBUG	getOAuthToken returning a cached token
1650117662831	Services.Common.RESTRequest	DEBUG	GET request to https://profile.accounts.firefox.com/v1/profile
1650117662831	Sync.SyncAuthManager	INFO	Getting sync key
1650117662832	Sync.SyncAuthManager	INFO	Getting a sync token from: https://token.services.mozilla.com/1.0/sync/1.5
1650117662832	Sync.SyncAuthManager	DEBUG	Getting a token using OAuth
1650117662832	FirefoxAccounts	DEBUG	getOAuthToken enter
1650117662832	FirefoxAccounts	TRACE	getCachedToken returning cached token
1650117662832	FirefoxAccounts	DEBUG	getOAuthToken returning a cached token
1650117662832	Services.Common.TokenServerClient	DEBUG	Beginning OAuth token exchange: https://token.services.mozilla.com/1.0/sync/1.5
1650117662832	Services.Common.RESTRequest	DEBUG	GET request to https://token.services.mozilla.com/1.0/sync/1.5
1650136843496	Sync.SyncScheduler	DEBUG	Network link status changed to "down". Offline?: false
1650136843496	Sync.SyncScheduler	DEBUG	Clearing sync triggers and the global score.
1650136843497	Sync.SyncScheduler	DEBUG	Network link status changed to "changed". Offline?: false
1650136843598	Sync.SyncScheduler	DEBUG	Woke from sleep.
1650136843598	Sync.SyncScheduler	DEBUG	Online, will sync in 2s.
1650136843598	Sync.SyncScheduler	DEBUG	Next sync in 2000 ms. (why=wake_notification)
1650136843599	Sync.SyncScheduler	DEBUG	Network link status changed to "changed". Offline?: false
1650136844910	Services.Common.RESTRequest	DEBUG	GET https://profile.accounts.firefox.com/v1/profile failed: 2152398878 - NS_ERROR_UNKNOWN_HOST
1650136844911	FirefoxAccounts	ERROR	Background refresh of profile failed: {"name":"FxAccountsProfileClientError","code":null,"errno":998,"error":"NETWORK_ERROR","message":"[Exception... \"NS_ERROR_UNKNOWN_HOST\"  nsresult: \"0x804b001e (NS_ERROR_UNKNOWN_HOST)\"  location: \"JS frame :: resource://services-common/rest.js :: onStopRequest :: line 488\"  data: no]"}
1650136844911	Services.Common.RESTRequest	DEBUG	GET https://token.services.mozilla.com/1.0/sync/1.5 failed: 2152398878 - NS_ERROR_UNKNOWN_HOST
1650136844912	Sync.SyncAuthManager	ERROR	Non-authentication error in _fetchTokenForUser: TokenServerClientNetworkError({"error":{}})(resource://services-common/tokenserverclient.js:39:36) JS Stack trace: TokenServerClientNetworkError@tokenserverclient.js:62:16
_tokenServerExchangeRequest@tokenserverclient.js:241:13
1650136844912	Sync.Status	DEBUG	Status.login: success.login => error.login.reason.network
1650136844912	Sync.Status	DEBUG	Status.service: success.status_ok => error.login.failed
1650136844912	Sync.SyncAuthManager	ERROR	Failed to fetch a token for authentication: TokenServerClientNetworkError({"error":{}})(resource://services-common/tokenserverclient.js:39:36) JS Stack trace: TokenServerClientNetworkError@tokenserverclient.js:62:16
_tokenServerExchangeRequest@tokenserverclient.js:241:13
1650136844912	FirefoxAccounts	TRACE	not checking freshness of profile as it remains recent
1650136845133	Sync.Resource	WARN	GET request to https://sync-1-us-west1-g.sync.services.mozilla.com/1.5/194602547/info/collections failed: [Exception... "The lookup of the hostname failed"  nsresult: "0x804b001e (NS_ERROR_UNKNOWN_HOST)"  location: "<unknown>"  data: no] No traceback available
1650136845133	Sync.Status	DEBUG	Status.sync: success.sync => error.login.reason.network
1650136845133	Sync.Status	DEBUG	Status.service: error.login.failed => error.sync.failed
1650136845134	Sync.ErrorHandler	ERROR	Sync encountered an error: [Exception... "The lookup of the hostname failed"  nsresult: "0x804b001e (NS_ERROR_UNKNOWN_HOST)"  location: "<unknown>"  data: no] No traceback available
1650136845134	Sync.SyncScheduler	DEBUG	Sync error count has exceeded 3; enforcing backoff.
1650136845135	Sync.SyncScheduler	DEBUG	Starting client-initiated backoff. Next sync in 8981245 ms.
1650136845135	Sync.SyncScheduler	DEBUG	Next sync in 8981245 ms. (why=client-backoff-schedule)
1650136845135	Sync.Telemetry	INFO	Early submission of sync telemetry due to changed IDs/NodeType
1650136845140	Sync.Service	DEBUG	Exception calling anonymous function: [Exception... "The lookup of the hostname failed"  nsresult: "0x804b001e (NS_ERROR_UNKNOWN_HOST)"  location: "<unknown>"  data: no] No traceback available
